CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
     - All existing users must add aureliumskills: 'true' to their config.yml's hook section.
     - added AureliumStatsLevelCondition
     - added AureliumSkillsLevelExperienceEvent
+- add DecentHolograms Compatibility
+    - requires DecentHolograms version 2.7.3 or newer for conditioned holograms.
```

Comment:
There were some weird problems with versioning. I will change it so that version 2.7.3 is required for it to work at all (since 2.7.2 doesn't work with hidden holograms, but that version added API for hidden holograms).